### DISCUSSION TITLE
https://www.r-bloggers.com/2024/12/how-to-replace-missing-values-in-r-a-comprehensive-guide/?utm_source=phpList&utm_medium=email&utm_campaign=R-bloggers-daily&utm_content=HTML

### DISCUSSION THREAD

**Discussion by giscus** [spsanderson/steveondata#19] [Comments]
# https://www.r-bloggers.com/2024/12/how-to-replace-missing-values-in-r-a-comprehensive-guide/?utm_source=phpList&utm_medium=email&utm_campaign=R-bloggers-daily&utm_content=HTML

Introduction Are you working with a dataset in R that has missing values? Don’t worry, it’s a common issue that every R programmer faces. In this in-depth guide, we’ll cover various techniques to effectively handle and replace missing values in...

https://www.r-bloggers.com/2024/12/how-to-replace-missing-values-in-r-a-comprehensive-guide/?utm_source=phpList&utm_medium=email&utm_campaign=R-bloggers-daily&utm_content=HTML

**Comment by Peter1960-21** [comment]
Please review Stef van Buuren on more appropriate methods of imputation. Mean or median substitution might be extremely wrong. SPSS has a method do use different values for missing codes. Using the module Categories, they can get different values better suited when missingness has different causes.

**Comment by spsanderson** [comment]
This was meant to be an illustration not an education that using the mean
value of a column is appropriate in any infinite number of cases.

The analyst must know their data.

Steven P Sanderson II, MPH
Amazon Author Profile
<https://www.amazon.com/stores/Steven-P.-Sanderson/author/B0CZ157DHK?ref_=pe_2466670_811284380&isDramIntegrated=true&shoppingPortalEnabled=true>
Book on Lulu <http://goo.gl/lmrlFI>
Personal Site <http://www.spsanderson.com>
